flexget/plugins/list/trakt_list.py

```diff
@@ -193,10 +193,10 @@ def items(self):
                     continue
                 entry = Entry()
                 if list_type == 'episode':
-                    entry['url'] = 'http://trakt.tv/shows/%s/seasons/%s/episodes/%s' % (
+                    entry['url'] = 'https://trakt.tv/shows/%s/seasons/%s/episodes/%s' % (
                         item['show']['ids']['slug'], item['episode']['season'], item['episode']['number'])
                 else:
-                    entry['url'] = 'http://trakt.tv/%s/%s' % (list_type, item[list_type]['ids'].get('slug'))
+                    entry['url'] = 'https://trakt.tv/%ss/%s' % (list_type, item[list_type]['ids'].get('slug'))
```

Comment:
@mfonville well... The variable `list_type` is used in two ways. One, and the most prevalent, is as the index or as the comparison to a dictionary index. The other, used only once, as part of this URL generation. 
I've opted to only make the URL generation slightly off and keep the majority the same.